modules/bluima_opennlp/pom.xml

```diff
@@ -45,5 +45,25 @@
         <artifactId>bluima_sentence_detector_pennbio</artifactId>
         <version>0.1</version>
     </dependency>
+    <dependency>
```

Comment:
It is currently used in the (deprecated) `OpenNlpHelper.getSentenceSplitter()` method.